pkg/core/typeurl/types.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package typeurl is Marushal For any type
+package typeurl
+
+import (
+	"encoding/json"
+	"path"
+	"reflect"
+	"sync"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/types"
+	"github.com/pkg/errors"
+)
+
+var registry map[reflect.Type]string
```

Comment:
can we add some comments for the registry and revRegistry variables ?

Reply:
ok
```
// registry store the reflectType and typeurl string map
var registry map[reflect.Type]string
// revRegistry store the typeurl string and reflectType map
var revRegistry map[string]reflect.Type
```